Automattic/simplenote-electron
# Fix scroll bar styles in Firefox

### Fix

This PR is to resolve #2640. 
In Firefox we were using the default styling for scroll bars in areas like the note list. In some cases this caused a scroll bar which didn't match the rest of the app.
Particularly on MacOS with System Preferences > General > Show scroll bars: set to anything other than `When scrolling` and using dark mode.

Firefox doesn't give as many options for styling scrollbars as Webkit browsers, there are only two properties.
- [scrollbar-color](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-color)
- [scrollbar-width](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width)

![Screen Shot 2021-02-08 at 10 34 18 AM](https://user-images.githubusercontent.com/1326294/107235162-c3a45780-69fa-11eb-86b9-c031e38dab66.png)
![Screen Shot 2021-02-08 at 10 34 37 AM](https://user-images.githubusercontent.com/1326294/107235174-c737de80-69fa-11eb-975f-cd7a5e18f159.png)

### Test

1. Using Firefox on MacOS with System Preferences > General > Show scroll bars: set to `Always`
2. Set theme to dark mode
3. Check scroll bars of note list.
4. Ensure they match theme well
5. Test in light mode as well.
6. Check Firefox on Windows and Linux to ensure they look ok as well.

### Release

- Update scrollbars
